src/contractTest/java/uk/gov/hmcts/reform/wataskconfigurationapi/consumer/ccd/CcdCasePactTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.reform.wataskconfigurationapi.consumer.ccd;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit5.PactTestFor;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.common.collect.Maps;
+import io.restassured.http.ContentType;
+import net.serenitybdd.rest.SerenityRest;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.wataskconfigurationapi.SpringBootContractBaseTest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CcdCasePactTest extends SpringBootContractBaseTest {
+
+    private static final String TEST_CASE_ID = "1607103938250138";
+    private static final String CCD_CASE_URL = "/cases/" + TEST_CASE_ID;
+
+    @Pact(provider = "ccd_data_store", consumer = "wa_task_configuration_api")
+    public RequestResponsePact ccdGetCasesId(PactDslWithProvider builder) {
```

Comment:
Since this is getting a case by id, can we rename this so it's more meaningful? E.g. executeCcdGetCasesByCaseId?

Reply:
naming convention?